### PR TITLE
[activitystream] Add back lost filter_backends

### PR DESCRIPTION
**Pull request by relrod**
These are needed so things like order_by works.

Signed-off-by: Rick Elrod <rick@elrod.me>

### CODE REVIEW

File: ansible_base/activitystream/views.py [hunk 1]
```diff
@@ -10,14 +10,37 @@
...
+    added_activity_stream_backend = False
+    for backend in default_filter_backends:
+        if backend.__name__ == 'FieldLookupBackend':
+            filter_backends.append(ActivityStreamFilterBackend)
+            added_activity_stream_backend = True
+        else:
+            filter_backends.append(backend)
+
+    if not added_activity_stream_backend:
+        filter_backends.append(ActivityStreamFilterBackend)
```
Comment (john-westcott-iv) [inline ansible_base/activitystream/views.py:31]: If I'm reading this right you are basically subbing `FieldLookupBackend` for `ActivityStreamFilterBackend` right?

Can we shorten this to just:
```python
if FieldLookupBackend in default_filter_backends:
    default_filter_backends.remove(FieldLookupBackend)
if ActivityStreamFilterBackend not in default_filter_backends:
    default_filter_backends.append(ActivityStreamFilterBackend)
```

Or something along those lines?

**Review by john-westcott-iv — APPROVED** [review]
Code reduction question but fine to merge as is.

File: ansible_base/activitystream/views.py (hunk 1 above)
Comment (relrod) [reply ansible_base/activitystream/views.py:31]: I wanted to retain order, just in case. I think DRF will apply the filtering backends consecutively and narrow the queryset further each time, so the order could matter in theory I think in some cases.
